### ISSUE TITLE
Incompatibility with numpy 1.16.0

### ISSUE THREAD

**Issue by balazsfabian** [Marcello-Sega/pytim#306]
```
Python 2.7.14 (default, Oct 12 2017, 15:33:51) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import pytim
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "build/bdist.macosx-10.10-x86_64/egg/pytim/__init__.py", line 8, in <module>
  File "build/bdist.macosx-10.10-x86_64/egg/pytim/willard_chandler.py", line 9, in <module>
  File "/Users/bfabian/.pyenv/versions/2.7.14/lib/python2.7/site-packages/skimage/__init__.py", line 167, in <module>
    from .util.dtype import (img_as_float32,
  File "/Users/bfabian/.pyenv/versions/2.7.14/lib/python2.7/site-packages/skimage/util/__init__.py", line 6, in <module>
    from .apply_parallel import apply_parallel
  File "/Users/bfabian/.pyenv/versions/2.7.14/lib/python2.7/site-packages/skimage/util/apply_parallel.py", line 8, in <module>
    import dask.array as da
  File "/Users/bfabian/.pyenv/versions/2.7.14/lib/python2.7/site-packages/dask/array/__init__.py", line 9, in <module>
    from .routines import (take, choose, argwhere, where, coarsen, insert,
  File "/Users/bfabian/.pyenv/versions/2.7.14/lib/python2.7/site-packages/dask/array/routines.py", line 256, in <module>
    @wraps(np.matmul)
  File "/Users/bfabian/.pyenv/versions/2.7.14/lib/python2.7/functools.py", line 33, in update_wrapper
    setattr(wrapper, attr, getattr(wrapped, attr))
AttributeError: 'numpy.ufunc' object has no attribute '__module__'
>>> import numpy
>>> numpy.__version__
'1.16.0'

```

**Comment by Marcello-Sega** [comment]
This looks like a cascade problem triggered by the changes in numpy 1.16. Updating dask to 0.20.0 solves the above problem, but another one in skimage.measure pops up, see  https://github.com/scikit-image/scikit-image/issues/3649

This has been fixed already in the master branch, but will be available only once scikit-image==1.14.2 is released. 

In the meanwhile, should we require numpy < 1.16 or should we enclose Willard-Chandler in a try/except until the new scikit-image is released? 

BTW, see also the release notes of numpy

https://github.com/numpy/numpy/releases

**Comment by Marcello-Sega** [comment]
we can make it even easier :) see #307

**Comment by stefanv** [comment]
skimage v0.14.2 has been released to address this.

**Comment by Marcello-Sega** [comment]
fixed by #307
